### PR TITLE
DOP-3270 - Add leaf icon to sidenav and fix buggy topnav positioning

### PR DESCRIPTION
**Pull request by carooobregon**
**Ticket**
[DOP-3270](https://jira.mongodb.org/browse/DOP-3270)

**Comments:**
More css changes
- Added the mongodb leaf to the desktop sidenav
- Menu icon in topnav had margin-right set up as auto which led to some awkward positioning in some edge cases like when the repo names were too long, etc. Changed it to a set value of 20px on margin-right to fix this issue.

### CODE REVIEW

File: frontend/src/components/SideNav.tsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import React, { useEffect, useState, useCallback } from 'react';
```
Comment (mmeigs) [inline frontend/src/components/SideNav.tsx:1]: nit: Don't know how we missed this, but could you delete these unused imports :)

File: frontend/src/components/TopNav.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ const NavButton = styled('button')`
   justify-content: flex-end;
   background-color: transparent;
   border: none;
-  margin-right: auto;
+  margin-right: 20px;
```
Comment (mmeigs) [inline frontend/src/components/TopNav.tsx:58]: Okay, this is the biggest nit I've maybe ever said??? But I for some reason love to have the "styled" components above the LeafyCSS style rules. I feel like the organization of it helps the code clarity. Could you change the order in both of these files? Also, let me know if you have a better idea or reasoning. Maybe it's just friday morning energy :)

**Review by mmeigs — COMMENTED** [review]
This looks perfect, Caro. Just left two teeny tiny nits.

File: frontend/src/components/TopNav.tsx (hunk 2 above)
Comment (mmeigs) [reply frontend/src/components/TopNav.tsx:54]: @carooobregon  Can you move the order in SideNav as well?

Comment (carooobregon) [reply frontend/src/components/TopNav.tsx:54]: yes sorry!

**Review by mmeigs — APPROVED** [review]
LGTM

**Pull request merged**